src/scandipwa/app/plugin/query/ConfigQuery.plugin.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @author Deniss Dubinins <denissd@scandiweb.com | info@scandiweb.com>
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import { PureComponent } from 'react';
+
+export class ConfigQueryPlugin extends PureComponent {
```

Comment:
You should not extend PureComponent with the plugin classes.
Also, FYI, I see that you actually don't need a plugin class here at all. We are shifting towards removing the classes from .plugin.js files when they don't serve some specific purpose, e.g. preserving the state.